### DISCUSSION TITLE
404 error whenever I click apply for organization

### DISCUSSION THREAD

**Discussion by KG-1510** [github/india#644] [Students]
After my profile got verified I tried to apply to projects, and upon clicking the "Apply" button it leads me to a 404 page, for all the projects. 

Is this a bug in the website or is it just me. Please help!

**Comment by rudranshsharma123** [comment]
it happens for me too

**Comment by Ruhi02** [comment]
I am also facing the same error

**Comment by 2010030555deepthi** [comment]
same
